1.segment_images/scripts/00.reslice_images.py

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# This notebook reslices the 3D volume into interpolated slices strictly for the purposes of visualization and segmentation.
+
+# In[1]:
+
+
+import argparse
+import pathlib
+
+import matplotlib.pyplot as plt
+
+# Import dependencies
+import numpy as np
+import skimage
+import tifffile
+import tqdm
+from skimage import io
+
+# check if in a jupyter notebook
+try:
+    cfg = get_ipython().config
+    in_notebook = True
+except NameError:
+    in_notebook = False
```

Comment:
This looks like a helpful function that could be useful in many notebooks. Consider promoting this to a utility function of some kind so as to reduce the amount of repeated code (esp. if it will or does appear in many notebooks).